### ISSUE TITLE
Missing file  benchmark/ips/stats/stats_metric

### ISSUE THREAD

**Issue by shveikus** [evanphx/benchmark-ips#102]
ruby version:  ```ruby 2.5.7p206 (2019-10-01 revision 67816) [x86_64-linux]```

gem version ```benchmark-ips (2.8.0)```

trace:
 ```Traceback (most recent call last):

	10: from profiling.rb:2:in `<main>'
	 9: from profiling.rb:2:in `require'
	 8: from /home/dmitry/work/some_project/lib/env.rb:3:in `<top (required)>'
	 7: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler.rb:174:in `require'
	 6: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `require'
	 5: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `each'
	 4: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:65:in `block in require'
	 3: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:88:in `rescue in block in require'
	 2: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:88:in `require'
	 1: from /home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/benchmark-ips-2.8.0/lib/benchmark/ips.rb:4:in `<top (required)>'
/home/dmitry/.rbenv/versions/2.5.7/lib/ruby/gems/2.5.0/gems/benchmark-ips-2.8.0/lib/benchmark/ips.rb:4:in `require': cannot load such file -- benchmark/ips/stats/stats_metric (LoadError)
```
When I have reinstalled with gem source to github repo everything worked fine.

**Comment by kbrock** [comment]
Pretty sure this is being addressed by #101

**Comment by nateberkopec** [comment]
Closed by #101
